sdk/communication/communication-call-automation/test/utils/recordedClient.ts

```diff
@@ -309,3 +326,16 @@ export async function getPhoneNumbers(recorder: Recorder): Promise<string[]> {
   }
   return phoneNumbers;
 }
+
+function sanitizeObject(obj: any, keysToSanitize: string[]) {
```

Comment:
not super familiar with JS but do we not have SanitizedHeaders to use? (.net https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/communication/Azure.Communication.CallAutomation/tests/Infrastructure/CallAutomationClientAutomatedLiveTestsBase.cs#L46) or do we have to do this for Events?

Reply:
sanitize options for js is provided during RecorderStartOptions - however, event payload is currently not being recorded by Recorder, but our custom functions (since events are not like regular request/response HTTP request)

you can see that for regular http request/responses, it uses proper sanitizer provided by Recorder (line 115)